src/options/view.rs

```diff
@@ -330,9 +330,39 @@ impl TimeFormat {
             "iso" => Ok(Self::ISOFormat),
             "long-iso" => Ok(Self::LongISO),
             "full-iso" => Ok(Self::FullISO),
-            fmt if fmt.starts_with('+') => Ok(Self::Custom {
-                fmt: fmt[1..].to_owned(),
-            }),
+            fmt if fmt.starts_with('+') => {
+                let mut lines = fmt[1..].lines();
+
+                // line 1 will be None when:
+                //   - there is nothing after `+`
+                // line 1 will be empty when:
+                //   - `+` is followed immediately by `\n`
+                let line_1_err_msg = "Custom timestamp format is empty, \
```

Comment:
I dont like the name of this var, can you name it something more meaning full such as empty timestamp format or anything like that

Reply:
Renamed to `empty_non_recent_format_msg` and `empty_recent_format_msg`.